src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
![image](https://github.com/mate-academy/layout_calendar/assets/59209337/f248f343-13f8-47fa-8cad-079d77a8da4b)

Reply:
Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)